tests/vmcheck/vmcheck-run.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+# Run all vmcheck tests using parallel.  If
+# VMCHECK_PARALLEL is set, it is a maximum number
+# jobs to run in parallel.
+#
+# FIXME: dedup this logic with tests/compose
+set -euo pipefail
+
+dn=$(cd $(dirname $0) && pwd)
+
+# Preparatory work; we have a helper binary
+make inject-pkglist
```

Comment:
This should already have been roped in by the make target, right?

Reply:
Yeah, but it's convenient to run the script directly too.